src/Switches.h

```diff
@@ -94,7 +95,35 @@ class SwitchOnCycleDuration {
 		}
 		g_cycleStartTime = nextCycleStartTime;
 	}
+
+	inline void execute(int ifd, int buffer_size) {
+		execute();
+	}
+
+};
+
+//=============================================================================
+class SwitchOnDummySend {
+public:
+	// dummy send between two cycles starting point and take starting point of next cycle
+
+	inline void execute(int ifd, int buffer_size) {
+		uint8_t buffer[buffer_size];
+		TicksTime nextCycleStartTime = g_cycleStartTime + g_pApp->m_const_params.cycleDuration;
+		while (!g_b_exit) {
+			if (TicksTime::now() >= nextCycleStartTime) {
+				break;
+			}
+
+			sendto(ifd, buffer, sizeof(buffer), MSG_SYN, (struct sockaddr*)& (g_fds_array[ifd]->server_addr), sizeof(struct sockaddr));
```

Comment:
better to create a sockperf level define, something like:
#define DUMMY_SEND_FLAG    MSG_SYN // Used by offload libraries to do egress path warm-up of caches. It is not in use by kernel. WARNING: it will actually end this packet on the wire.